include/seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp

```diff
@@ -477,7 +478,7 @@ void stream_alignment(debug_stream_type<char_t> & stream, alignment_t const & al
         {
             // write alignment bars
             stream << '\n' << std::setw(8) << "";
-            std::ranges::for_each(ranges::zip_view(previous_seq, aligned_seq) | views::slice(begin_pos, end_pos),
+            std::ranges::for_each(views::zip(previous_seq, aligned_seq) | views::slice(begin_pos, end_pos),
                              [&stream] (auto && ch) { stream << (get<0>(ch) == get<1>(ch) ? '|' : ' '); });
```

Comment:
indentation

Reply:
Of what excactly? The lambda expression? I can indent it, but it's not something I touched in this PR.